packages/core/src/components/text/Text.module.css

```diff
@@ -0,0 +1,70 @@
+.text {
+  /* Theme vars */
+  --swui-text-color: var(--swui-text-primary-color);
+  --swui-text-font-family: var(--swui-font-primary);
+  --swui-text-font-weight: var(--swui-font-weight-text);
+  --swui-text-letter-spacing: -0.34px;
+
+  /* Current */
+  --current-color: var(--swui-text-color);
+  --current-font-size: var(--swui-font-size-medium);
+  --current-line-height: var(--swui-line-height-medium);
+  --current-font-weight: var(--swui-text-font-weight);
+  --current-letter-spacing: var(--swui-text-letter-spacing);
+
+  /* Styling */
+  font-size: var(--current-font-size);
+  line-height: var(--current-line-height);
+  color: var(--current-color);
+  letter-spacing: var(--current-letter-spacing);
+  font-family: var(--swui-text-font-family);
+  font-weight: var(--current-font-weight);
+
+  &.standard {
+  }
+
+  &.bold {
+    --current-font-weight: var(--swui-font-weight-text-bold);
+  }
+
+  &.caption {
+    --current-font-size: var(--swui-font-size-small);
+    --current-line-height: var(--swui-line-height-small);
+    --current-color: var(--lhds-color-ui-700);
```

Comment:
Why is this explicit

Reply:
I added theme variables for the shared fields only. Maybe I should add the colors, so they can be easily overridden.